packages/components/src/Modal/createPortal.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactNode, useEffect } from "react";
+import ReactDOM from "react-dom";
+
+interface CreatePortalProps {
+  children: ReactNode | ReactNode[];
+}
+
+export function CreatePortal({ children }: CreatePortalProps) {
+  return <Portal>{children}</Portal>;
+}
+
+interface PortalProps {
+  children: ReactNode | ReactNode[];
+}
+
+export function Portal({ children }: PortalProps) {
+  const modalRoot = document.body.appendChild(document.createElement("div"));
+  const tempVar = document.createElement("div");
```

Comment:
I'm trying to decide if there is any benefit to creating a temp div for each portal… I'm not thinking of any immediately I think lets simplify this part for now?